src/test/java/org/sagebionetworks/bridge/exporter/synapse/SynapseStatusTableHelperTest.java

```diff
@@ -40,17 +42,18 @@
     private static final String SYNAPSE_TABLE_NAME = "test-study-status";
 
     private String synapseTableId;
+    private ExportWorkerManager mockManager;
+    private SynapseHelper mockSynapseHelper;
+    private SynapseStatusTableHelper statusTableHelper;
 
     @BeforeMethod
     public void before() {
         // clear helper vars, because TestNG doesn't always do that
         synapseTableId = null;
     }
 
-    @Test
-    public void test() throws Exception {
-        // mock manager
```

Comment:
Don't delete these comments. Code is too hard to read if it's just a bunch of code and no comments.